ldap_validator/test/test_data/yaml_data/testsitedefault_assertion.yml

```diff
@@ -0,0 +1,22 @@
+# Internal SAS site defaults
+#
+# For instructions on how to use this file, see:
+# http://documentation.sas.com/?softwareId=calcdc&softwareVersion=production.a&softwareContextId=sitedefault
```

Comment:
Following this link you see displayed there a warning about not sharing internal links with customers.   Why is this link needed here?

Reply:
I don't know that it is needed. It can be deleted, but will discuss.

Reply:
Will be removed with upcoming push.